starlink_grpc.py

```diff
@@ -549,7 +540,10 @@ def __init__(self, e, *args, **kwargs):
 
 
 class UnwrappedHistory:
-    """Empty class for holding a copy of grpc history data."""
+    """Class for holding a copy of grpc history data."""
+
+    unwrapped: bool
+    current: Any
```

Comment:
Is this really necessary? The design intent of this class is for it to be a dynamically typed replacement for `device_pb2.DishGetHistoryResponse` with the addition of the `unwrapped` attribute, and that it be largely opaque outside this module.

That being said, there's no reason it _couldn't_ fully specify the type info, but in that case, you should also add all the attributes listed in `HISTORY_FIELDS` (as `Sequence[float]`, also `current` is an `int`). I just didn't see the point of doing that, so I just treated it as a dynamic type in `concatenate_history`.

Reply:
It's not really necessary, but it does fix syntax highlighting for internal code. I can revert it if you'd like

Reply:
Alright, I have removed `current` from it altogether. I tried changing the type to `int` but my IDE still showed `Any`, so I figured it wasn't worth it just yet